src/GafferScene/ShaderAssignment.cpp

```diff
@@ -51,6 +51,9 @@ namespace
 const InternedString g_oslShader( "osl:shader" );
 const InternedString g_oslSurface( "osl:surface" );
 
+const char *g_oslPrefix( getenv( "SHADER_ASSIGNMENT_OSL_PREFIX" ) );
```

Comment:
The established pattern (line 82) would be `GAFFERSCENE_SHADERASSIGNMENT_...`. Let's use that.